### PR TITLE
Fixing some code errors and cleaning out old stuff

### PR DESCRIPTION
**Pull request by VSEphpbb**
Some incredibly outdated docs have been removed and/or updated.

Also, I’m shocked Titania actually works at all. There are so many inspection errors, and so much spagetti code I fixed only the simplest and most obvious errors that shouldn’t appear to break anything (all the other things I’d like to fix will probably break something unforeseen).

### CODE REVIEW

File: controller/ucp/subscriptions.php [hunk 1]
```diff
@@ -184,7 +184,7 @@ protected function display_sections()
 					if (!$row['contrib_id'])
 					{
 						$this->delete_subscription($row['watch_object_type'], $row['watch_object_id'], false);
-						continue;
```
Comment (paul999) [inline controller/ucp/subscriptions.php]: Please make sure this works on 7.3

File: docs/SETUP.md [hunk 2]
```diff
@@ -2,13 +2,27 @@
 
 This is a quick setup guide to walk you through the process of installing and configuring Titania.
 
+## Requirements
+
+1. phpBB 3.1.x, 3.2.x or 3.3.x
```
Comment (paul999) [inline docs/SETUP.md]: you are sure?

File: controller/ucp/subscriptions.php (hunk 1 above)
Comment (VSEphpbb) [reply controller/ucp/subscriptions.php]: Yes. In fact 7.3 would issue a warning here saying we should be using `continue 2`

File: docs/SETUP.md (hunk 2 above)
Comment (VSEphpbb) [reply docs/SETUP.md]: Just going by Titania's composer.

https://github.com/phpbb/customisation-db/blob/3.2.x/composer.json#L74

File: controller/ucp/subscriptions.php (hunk 1 above)
Comment (paul999) [reply controller/ucp/subscriptions.php]: Ok, just wanted to be sure :)

**Pull request merged**